tests/test_extensions/test_ext_apidoc.py

```diff
@@ -56,19 +62,16 @@ def test_pep_0420_enabled(make_app, apidoc):
     assert (outdir / 'a.b.e.rst').is_file()
     assert (outdir / 'a.b.x.rst').is_file()
 
-    with open(outdir / 'a.b.c.rst', encoding='utf-8') as f:
-        rst = f.read()
-        assert "automodule:: a.b.c.d\n" in rst
-        assert "automodule:: a.b.c\n" in rst
+    rst = (outdir / 'a.b.c.rst').read_text(encoding='utf-8')
```

Comment:
How come this was done if you ignore FURB101?

Reply:
unsure...
but it's definitely an improvement

Reply:
i think i started applying FURB101 and decided that changing from str to Path needs its own PR.
but i think these changes stand on their own and should be kept